meetings/2019-04-23.md

```diff
@@ -1,9 +1,8 @@
 # TSC Meeting April 23, 2019 @12PM MST
 
-## [Zoom Link](???)
+## [Zoom Link](https://zoom.us/j/619878062)
 
 ## Agenda
-- Organizing a planetary science software group (https://github.com/USGS-Astrogeology/TSC/issues/28).
```

Comment:
This meeting is only for Andrew's membership, not this item. This will be discussed further in May.

Reply:
Andrew's membership and otherwise extending conversation, I don't think we'll take any other votes or do anything like that.